src/accounts.js

```diff
@@ -1,32 +1,38 @@
-"use strict";
+'use strict';
 
+const { flatten } = require('lodash');
+const Promise = require('bluebird');
 const api = require('./api');
 
+function getPath(bmId) {
+  if (!bmId) return 'ma/adaccounts';
```

Comment:
> ma
Also should this be prefixed with a slash as well?

Reply:
hot on the merge. i fixed in master though